tools/catkubeletconfmap/catkubeletconfmap.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
license statement?

Reply:
good point, adding the license boilerplate. Silly me just forgot.

Reply:
let's add them all in one go: https://github.com/openshift-kni/numaresources-operator/pull/673